packages/template-retail-react-app/package.json

```diff
@@ -83,7 +83,7 @@
         },
         {
             "path": "build/vendor.js",
-            "maxSize": "270 kB"
+            "maxSize": "280 kB"
```

Comment:
Is this bump still required?

Reply:
Yes it is, the vendor now is around 276kb.
![image](https://user-images.githubusercontent.com/52219283/190828566-ad84b42d-68bf-4860-a260-ab792edc6fff.png)